### PR TITLE
Add new design changes

### PR DESCRIPTION
**Pull request by aavrug**
* **Please check if the PR fulfills these requirements**

- [ ] Screenshots for changes (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: themes/inpycon2020/templates/base.html [hunk 1]
```diff
@@ -86,7 +146,7 @@
 	</div>
       </div>
     </div>
-  </footer>
+  </footer> -->
```
Comment (ananyo2012) [inline themes/inpycon2020/templates/base.html:149]: Please remove the comment

Comment (aavrug) [reply themes/inpycon2020/templates/base.html]: Done

**Comment by ananyo2012** [comment]
> Just a quick comment, I think having `PyCon India 2020 Online` in the same line on the banner would look better. We don't have any real estate crunch.
> 
> @aavrug Can you put up a mock design with that so that we can get a preview?

Ah I think that won't look good. Still I will get the mockup done by Siddharth who did the original design to compare

**Comment by aavrug** [comment]
> @aavrug Can you update the navbar links with that from the PyCon India Website ? The links are not correct now. Also add a Home button pointing to PyCon India website homepage in navbar

Where exactly you need the Home button, either give some markup or some rough sketch?

**Comment by ananyo2012** [comment]
> > @aavrug Can you update the navbar links with that from the PyCon India Website ? The links are not correct now. Also add a Home button pointing to PyCon India website homepage in navbar
> 
> What about the speakers link and where exactly you need the Home button, either give some markup or some rough sketch?

Put the Home in the navbar. It should be the first item in the navbar. So navbar will contain these items

Home -> https://in.pycon.org/2020/
About -> https://in.pycon.org/2020/#about
CFP -> https://in.pycon.org/2020/#cfp
Blog -> https://in.pycon.org/blog
Sponsor -> https://in.pycon.org/2020/#sponsor
CoC -> https://in.pycon.org/2020/coc.html
Blog ->

**Comment by ananyo2012** [comment]
@palnabarun Here is the design change you requested https://www.figma.com/file/l3fowoKJ1OAyXRd7j095r5/PyconIND-Assets?node-id=383%3A2

I like the previous one though

**Comment by aavrug** [comment]
> > > @aavrug Can you update the navbar links with that from the PyCon India Website ? The links are not correct now. Also add a Home button pointing to PyCon India website homepage in navbar
> > 
> > 
> > What about the speakers link and where exactly you need the Home button, either give some markup or some rough sketch?
> 
> Put the Home in the navbar. It should be the first item in the navbar. So navbar will contain these items
> 
> Home -> https://in.pycon.org/2020/
> About -> https://in.pycon.org/2020/#about
> CFP -> https://in.pycon.org/2020/#cfp
> Blog -> https://in.pycon.org/blog
> Sponsor -> https://in.pycon.org/2020/#sponsor
> CoC -> https://in.pycon.org/2020/coc.html

It's done

**Comment by ananyo2012** [comment]
@aavrug There is a issue with the logo . I can't see in the banner. The size seems to be zero. Can you check what's the issue ?

**Comment by aavrug** [comment]
> @aavrug There is a issue with the logo . I can't see in the banner. The size seems to be zero. Can you check what's the issue ?

I tried to replicate the issue but I don't think there is any issue.

**Comment by ananyo2012** [comment]
Will merge this by 9pm today if no other review comes in.

**Pull request merged**